### PR TITLE
[blog] Added article about approach to developing application

### PR DESCRIPTION
**Pull request by rkarlovic**
Added article about approaching developing of application based on book club discussion we had on Discord

### CODE REVIEW

File: content/rkarlovic/posts/Where should you start building your application/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+{
+  title: "Where should you start building your application",
+  description: "Talking about where you should start building your full stack application",
+  published: '2024-08-10',
+  tags: ["meta"],
```
Comment (fennifith) [inline content/rkarlovic/posts/Where should you start building your application/index.md:6]:
```suggestion
  tags: ["opinion", "webdev"],
```

Suggesting some different tags here :)

File: content/rkarlovic/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  name: "Ratomir Karlović",
+  firstName: "Ratomir",
+  lastName: "Karlović",
+  description: "Fullstack developer",
+  socials: {
+    github: "rkarlovic",
+    linkedin: "https://www.linkedin.com/in/ratomir-karlovi%C4%87-323bb6116/"
+  },
+  profileImg: "./seagullB.png",
+  roles: [ "authors" ]
```
Comment (fennifith) [inline content/rkarlovic/index.md:12]:
```suggestion
  roles: [ "author" ]
```

**Review by fennifith — COMMENTED** [review]
This looks great!

`"meta"` isn't actually a supported tag atm so I've suggested some that might fit the post ([tags.json](https://github.com/playfulprogramming/playfulprogramming/blob/main/content/data/tags.json) defines the possible options).

Can be merged once those changes are made :)

**Review by fennifith — APPROVED** [review]

File: content/rkarlovic/posts/Where should you start building your application/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+{
+  title: "Where should you start building your application",
+  description: "Talking about where you should start building your full stack application",
+  published: '2024-08-10',
```
Comment (crutchcorn) [inline content/rkarlovic/posts/Where should you start building your application/index.md:5]:
```suggestion
  published: '2024-08-21',
```

File: content/rkarlovic/posts/Where should you start building your application/index.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  tags: ["opinion", "webdev"],
+  license: 'cc-by-4'
+}
+---
+
+On our Discord channel we have started "book club" where someone suggest article for that week, and then we have discussion around it.
```
Comment (crutchcorn) [inline content/rkarlovic/posts/Where should you start building your application/index.md:11]:
```suggestion
[On our Discord channel](https://discord.gg/FMcvc6T) we have started "book club" where someone suggest article for that week, and then we have discussion around it.
````

**Pull request merged**